common/middleware/collection/set-date-range.js

```diff
@@ -12,6 +12,7 @@ function setDateRange(req, res, next, date) {
   }
 
   req.params.dateRange = dateRange
+  req.dateRange = dateRange // https://github.com/expressjs/express/issues/2911
```

Comment:
tl;dr - `req.params` is not modifiable as it's parsed out of the route. I'm not sure why it works in other places in the application.